.eslintrc.json

```diff
@@ -35,6 +36,7 @@
           { "name": ["test", "only"], "message": "Calls to test.only is not allowed" },
           { "name": ["describe", "only"], "message": "Calls to describe.only is not allowed" }
         ],
+        "license-header/header": ["error", "./license-header-aca.txt"],
```

Comment:
Not sure if that is only Intellij problem but I can see some issues with that: 
First: 
<img width="1167" alt="image" src="https://user-images.githubusercontent.com/115619721/231384655-4eae6fd7-e731-4cab-97cf-21f711ecc01d.png">

To fix that I need to make changes like: 
<img width="1171" alt="image" src="https://user-images.githubusercontent.com/115619721/231384792-13e75a50-5c1a-421e-a670-5cd1f06fc7d3.png">


And then I'm getting this issue: 
<img width="1170" alt="image" src="https://user-images.githubusercontent.com/115619721/231384894-127c1408-95e5-4b4e-ae89-6c8eafb35ff9.png">


Which I can fix in that way: 
<img width="1213" alt="image" src="https://user-images.githubusercontent.com/115619721/231385038-02f21109-e762-4c7f-878c-ff412237a4c0.png">


Do we want to fix that and use above fixes for all files which required?

Reply:
Interesting, I don't such problem in Intllij. Did you run 'npm ci', I installed a new plugin

Reply:
Yup, before installing plugin I had different issue. After installing it, old issue gone but I can see now these issues from above screenshots. Here is installed plugin in node modules: 
<img width="618" alt="image" src="https://user-images.githubusercontent.com/115619721/231389580-807d6cbf-8bb4-491e-8cd4-6fec73778be0.png">

You see I have installed it

Reply:
Wondering if we can correct that if that does not break anything for VSC?

Reply:
BTW which Intellij did you test? Ultimate or Community? It may be related with Intellij's version. Also I installed the newest version from 2023

Reply:
I have WebStorm 2022.1.3

Reply:
Ok so it may be reason why it works for you. Intellij may treat that in different way. Would you like to fix it for intellij too? Unless it makes any issues for VSC/WebStorm then we may live with that problem for Intellij

Reply:
If I apply changes from your screenshot I get parsing errors. MB it is some cache problems with your IDE?

Reply:
@AleksanderSklorz @nikita-web-ua I have spent a lot of time with the license headers and IntelliJ has problems with relative paths for eslint, so the fix for the monorepo was embedding the header into the rule like this:

```
"license-header/header": ["error", [
                  "/*",
                  " * Copyright © 2005 - 2021 Alfresco Software, Ltd. All rights reserved.",
                  " *",
                  " * License rights for this program may be obtained from Alfresco Software, Ltd.",
                  " * pursuant to a written agreement and any use of this program without such an",
                  " * agreement is prohibited.",
                  " */"
                ]],
```

Reply:
@DenysVuika thanks, I'll change it to follow this pattern